### PR TITLE
Implicit host tags on KVM

### PR DESCRIPTION
**Pull request by weizhouapache**
This is the doc PR for https://github.com/apache/cloudstack/pull/8929

----
📚 Documentation preview 📚: https://cloudstack-documentation--392.org.readthedocs.build/en/392/

### CODE REVIEW

**Comment by weizhouapache** [comment]
Update
https://cloudstack-documentation--392.org.readthedocs.build/en/392/adminguide/host_and_storage_tags.html#host-tags

**Comment by weizhouapache** [comment]
@shwstppr 
I added some lines in this PR. you can find the `flexible tags` section on same page
we could add more info , do you have advice ?

**Review by shwstppr — APPROVED** [review]
@weizhouapache functionality-wise changes look good. Others can review for language/presentation. @rohityadavcloud @DaanHoogland @andrijapanicsb

File: source/adminguide/host_and_storage_tags.rst [hunk 1]
```diff
@@ -23,6 +23,18 @@ Host tags
 ---------
 Host tags are responsible for directing VMs to compatible hosts. They are validated with the host tags informed in the compute offerings or in the system offerings.
 
+There are two types of host tags:
+
+- (Explicit) host tags: the host tags are managed by CloudStack. Users can set, update, and delete host tags via CloudStack API or GUI.
```
Comment (andrijapanicsb) [inline source/adminguide/host_and_storage_tags.rst:28]:
```suggestion
- (Explicit) host tags: the host tags are managed by CloudStack. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
```

Comment (weizhouapache) [reply source/adminguide/host_and_storage_tags.rst]: thanks @andrijapanicsb , committed

**Comment by andrijapanicsb** [comment]
There is no explanation of the implicit host tags here, however it's given in the PR description for the feature. 
We need to add it to the docs - at the very least - copy the functionality PR description into this doc PR.

**Comment by JoaoJandre** [comment]
@weizhouapache It would be good to have a note explaining that flexible tags must always be explicit

File: source/adminguide/host_and_storage_tags.rst [hunk 2]
```diff
@@ -23,6 +23,18 @@ Host tags
 ---------
 Host tags are responsible for directing VMs to compatible hosts. They are validated with the host tags informed in the compute offerings or in the system offerings.
 
+There are two types of host tags:
+
+- (Explicit) host tags: the host tags are managed by CloudStack. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
```
Comment (weizhouapache) [inline source/adminguide/host_and_storage_tags.rst:28]:
```suggestion
- (Explicit) host tags: the host tags are managed by CloudStack, including the flexible host tags. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
```

**Comment by weizhouapache** [comment]
> @weizhouapache It would be good to have a note explaining that flexible tags must always be explicit

added @JoaoJandre

File: source/adminguide/host_and_storage_tags.rst [hunk 3]
```diff
@@ -23,6 +23,18 @@ Host tags
 ---------
 Host tags are responsible for directing VMs to compatible hosts. They are validated with the host tags informed in the compute offerings or in the system offerings.
 
+There are two types of host tags:
+
+- (Explicit) host tags: the host tags are managed by CloudStack, including the flexible host tags. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
+- Implicit host tags: the host tags are not managed by CloudStack API.
```
Comment (weizhouapache) [inline source/adminguide/host_and_storage_tags.rst:29]:
```suggestion
- Implicit host tags: the host tags are not managed by CloudStack API. Cloud operator cannot use both implicit host tags and flexible host tags.
```

File: source/adminguide/host_and_storage_tags.rst [hunk 4]
```diff
@@ -23,6 +23,18 @@ Host tags
...
+
+.. note::
+   Currently the implicit host tags are only configurable on KVM hosts.
```
Comment (weizhouapache) [inline source/adminguide/host_and_storage_tags.rst:32]:
```suggestion
   Cloud operator can easily set the implicit host tags per host based on server configurations,
   for example hardware devices (GPU type, disk type, network card type, etc).
   Currently the implicit host tags are only configurable on KVM hosts. 
```

**Review by andrijapanicsb — APPROVED** [review]
LGTM

**Pull request merged**